src/components/FoodForm.js

```diff
@@ -1,71 +1,151 @@
 import {nanoid} from 'nanoid';
 import {useState} from 'react';
+import styled from 'styled-components';
 
+import {StyledResults} from './styled/StyledContainer';
 import {StyledDoneLabel} from './styled/StyledDoneLabel';
 import {StyledForm} from './styled/StyledForm';
+import {StyledImageCenter} from './styled/StyledImage';
 import {StyledInput} from './styled/StyledInput';
+import {StyledAtag} from './styled/StyledLink';
 import {StlyedList} from './styled/StyledList';
-import {StyledListItems} from './styled/StyledListItems';
+import {StyledListItems, StyledFetchItems} from './styled/StyledListItems';
+import {StyledBlackButton, StyledWhiteButton} from './styled/StyledNavButton';
+import {StyledCalories} from './styled/StyledNew';
+
+const StyledFetchButtons = styled.form`
```

Comment:
The naming and the html element do **not** match together. 🚫 Is this a button or a form element? Both together do not work.